webpack/scenes/Subscriptions/components/SubscriptionsTable/components/Table.js

```diff
@@ -34,9 +34,12 @@ const Table = ({
     toggle: ({ rowData }) => toggleSubscriptionGroup(rowData.product_id),
   };
 
+  const collapsibleGroup = () =>
```

Comment:
I think the `some` function (returns a boolean) might be advantageous here: https://www.w3schools.com/jsref/jsref_some.asp

`const collapsibleGroup = Object.values(groupedSubscriptions).some(v => v.subscriptions.length > 1);`